pcsx2/GameList.cpp

```diff
@@ -839,7 +839,7 @@ bool GameList::GetSerialAndCRCForFilename(const char* filename, std::string* ser
 
 std::string GameList::GetPlayedTimeFile()
 {
-	return Path::Combine(EmuFolders::Settings, "playtime.dat");
+	return Path::Combine(EmuFolders::Logs, "playtime.dat");
```

Comment:
This doesn't seem like an appropriate place to put this file.

Reply:
I thought that since it tracks gametime, it is logging some information, but okay. Where would you want this to be?

Reply:
> I thought that since it tracks gametime, it is logging some information, but okay. Where would you want this to be?

By that logic, so is every settings and save file. The original place is ideal and so maybe making a similar feature to Dolphin to redirect the settings folder would be a better solution than adding it into the Logs folder and hoping the user relocates it with the new setting.

Reply:
Alright, the original location will be kept.